core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -2742,6 +2741,27 @@ protected static CharSequence maybeExpectToken(GenericLexer lexer, CharSequence
         return tok;
     }
 
+    // returns true if it dropped the table, false otherwise (or throws exception)
+    protected boolean dropTable(
```

Comment:
nit: would be nice to add this method to `SqlCompilerWrapper` in `SqlCompilerImplTest`, so that we verify that this method is not renamed or accessor modifier is changed in future.

Reply:
did not notice there was a test like that.
let me add it.

Reply:
added in f137c50a2c091deb05b68fbb68082b12c2ab3200